.github/workflows/ci.yml

```diff
@@ -71,6 +71,7 @@ jobs:
     uses: ./.github/workflows/check.yml
     with:
       test_repository: ${{ github.repository }}
+      test_ref: ${{ github.ref }}
```

Comment:
WAIT. I merged prematurely. I think this should be the commit hash instead of the ref. So this should be `${{ github.sa }}

Reply:
https://github.com/terminusdb/terminusdb/pull/1179